mantidproject/mantid
# Reduce Pylint warning in Reflectometer files

This PR attempts to reduce the Pylint warnings produced by the files in `scripts/interfaces/reduction_gui/widgets/reflectometer`, whilst maintaining their current functionality.
- [x] `launch_peak_back_selection_1d.py`
- [x] `refl_data_simple.py`
- [x] `base_ref_reduction.py`

**To test:**
- Make sure that all builds pass
- Ensure that code changes are clean and make sense
- Ensure that the files in question have a reduced amount of warnings on the Pylint Violations list.



Fixes #15169.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
